composer.json

```diff
@@ -157,6 +157,7 @@
     "conflict": {
         "behat/transliterator": "< 1.3.0",
         "doctrine/lexer": ">= 3.0.0",
+        "doctrine/doctrine-bundle": "2.13.1 || 2.13.x-dev || 2.14.x-dev",
```

Comment:
remove 2.13.x-dev and 2.14.x-dev when https://github.com/doctrine/DoctrineBundle/pull/1841 is merged